docs/metrics/cohen-kappa.md

```diff
@@ -0,0 +1,114 @@
+# Cohen's Kappa
+
+Cohen's Kappa is a statistical metric that evaluates the reliability of an algorithm's predictions with similar goals
+as [accuracy](./accuracy.md) or [F<sub>1</sub>-score](./f1-score.md). However, the Cohen Kappa score magnifies the
+advantage a classifier has over a random classifier based on class frequencies. The Kappa statistic is a robust metric,
+particularly when there is significant class imbalance for binary and multiclass classification problems, which
+extends to more complicated ML tasks such as object detection.
+
+## Implementation Details
+
+Cohen's Kappa coefficient (\(\kappa\)) is calculated by comparing the observed agreement
+([true positives and true negatives](./tp-fp-fn-tn.md)) between the model inferences and the ground truths against
+expected classifications by random chance based on the marginal frequencies of each class.
+
+The formula for Cohen's Kappa is:
+
+$$
+\kappa = \frac{P_o - P_e}{1 - P_e}
+$$
+
+where:
+
+- \(P_o\) is the `observed agreement`
+- \(P_e\) is the hypothetical probability of `chance agreement`
+
+??? question "Understanding \(P_o\) and \(P_e\)"
```

Comment:
This content seems important to understand the original formula. What do you think of using a callout that uncollapses the content by default?